README.md

```diff
@@ -142,6 +142,15 @@ try {
 await conn.close();
 }
 ```
+Note:
+ - The current TcpStream implementation emits per line of data received. That is it reads data from the socket until a new line is reached, at which time it will emit the string line received (by calling onData(data,null).
```

Comment:
as a temporary workaround to `\n` problem (if the last chunk of received data doesn't contain newline), you could set a timeout of, say, 5 sec, and if buffer is still not empty (because there is no newline so `onData` is not emitted), forcefully emit `onData`. ofcourse, every new arrival of data to buffer resets the timeout